### PR TITLE
Fix `path` warnings

### PR DESCRIPTION
**Pull request by alyohea**
This pr should fix warnings (first two lines)

Screenshot 2022-02-04 at 13 16 49

### CODE REVIEW

**Comment by yohanboniface** [comment]
Ah, see my comment [here](https://github.com/peopledoc/django-agnocomplete/commit/bcb18379322dab8949469618738bc20de73d3824#r66016509).

I'd say that the correct fix is to keep using `path`, but with an [internal converter](https://docs.djangoproject.com/en/3.2/topics/http/urls/#path-converters), something like `<klass:slug>` seems to do the job, but I've not tested.

**Comment by yohanboniface** [comment]
The other way around, sorry: `'<slug:klass>/'`

**Review by nnachit — APPROVED** [review]
LGTM

File: agnocomplete/urls.py [hunk 1]
```diff
@@ -5,9 +5,6 @@
 from .views import AgnocompleteView, CatalogView
 
 urlpatterns = [
-    path(
-        r'^(?P<klass>[-_\w]+)/$',
-        AgnocompleteView.as_view(),
-        name='agnocomplete'),
-    path(r'^$', CatalogView.as_view(), name='catalog'),
+    path('<klass>/', AgnocompleteView.as_view(), name='agnocomplete'),
```
Comment (nnachit) [inline agnocomplete/urls.py:8]: Just a question. Why don't we use `re_path()` instead of `path()`

Comment (alyohea) [reply agnocomplete/urls.py:8]: Hm i really don't know, but i saw the comments here https://github.com/peopledoc/django-agnocomplete/commit/bcb18379322dab8949469618738bc20de73d3824

But request management already migrated to the Django 3 and didn't faced any problems with previous code (except the warning)

**Comment by alyohea** [comment]
Tested on my local machine, it works fine

**Comment by KeserOner** [comment]
So can we merge and release?

**Pull request merged**